install.sh

```
@@ -0,0 +1,1349 @@
+        harbor_extra_vars="-e containerd_insecure_registries=['\"${harbor_registry}\"']"
+        echo -e "  ${GREEN}Harbor Registry:${NC} ${harbor_registry} (자동 주입)"
+        echo ""
+    fi
+
+    if [ "$DEPLOY_MODE" = "offline" ]; then
+        # Offline: extra-vars + offline.yml + harbor registry
+        ansible-playbook \
+            -i "${customer_dir}/hosts.yml" \
+            -e "@${customer_dir}/extra-vars.yml" \
+            -e "@${customer_dir}/offline.yml" \
+            $harbor_extra_vars \
```

Shellcheck (0.11.0)

[warning] Quotes/backslashes in this variable will not be respected.

(SC2090)